src/main/java/com/project/parkinglot/model/mapper/parking_area/ParkingAreaCreateRequestToParkingAreaEntityMapper.java

```diff
@@ -0,0 +1,26 @@
+package com.project.parkinglot.model.mapper.parking_area;
+
+import com.project.parkinglot.model.dto.request.parking_area.ParkingAreaCreateRequest;
+import com.project.parkinglot.model.entity.ParkingAreaEntity;
+import com.project.parkinglot.model.mapper.BaseMapper;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface ParkingAreaCreateRequestToParkingAreaEntityMapper extends BaseMapper<ParkingAreaCreateRequest, ParkingAreaEntity> {
+
+    @Override
+    // @Mapping(source = "priceListRequest.name", target = "priceListEntity.name")
+    @Mapping(source = "priceListRequest.prices", target = "priceListEntity.priceEntities")
+    ParkingAreaEntity map(ParkingAreaCreateRequest source);
+
+    /**
+     * Initializes the mapper.
+     *
+     * @return the initialized mapper object.
+     */
+    static ParkingAreaCreateRequestToParkingAreaEntityMapper initialize() {
+        return Mappers.getMapper(ParkingAreaCreateRequestToParkingAreaEntityMapper.class);
+    }
```

Comment:
Buradan sonra kodun daha okunabilirliği için bir satır boşluk bırakalım.

Reply:
tamamdır